src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,21 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import mate.academy.spring.validation.Email;
+import mate.academy.spring.validation.ValidPassword;
+
+@ValidPassword(
+        passwordField = "password",
+        repeatPasswordField = "repeatPassword"
+)
 public class UserRequestDto {
+    @Email
     private String email;
+    @NotNull
```

Comment:
Better add a check for null value to your password validator aadn remove this annotation